### PR TITLE
adding RabbitMQ's rpm repo to install updated version of rabbitmq & erlang

### PR DESCRIPTION
**Pull request by romainrbr**
https://github.com/sensu/sensu-ansible/issues/102

### CODE REVIEW

**Comment by jaredledvina** [comment]
Hey @romainrbr , I just merged it https://github.com/sensu/sensu-ansible/pull/129, can you rebase your branch off of the latest master, I think, that'll get Github to kick off the TravisCI build to verify this change.

**Comment by romainrbr** [comment]
Seems to be good right now, apparently yum doesn't handle well having multiple base url.

File: tasks/CentOS/rabbit.yml [hunk 1]
```diff
@@ -4,10 +4,32 @@
 
   - include_vars: "{{ ansible_distribution }}.yml"
 
+  - name: Add RabbitMQ's repo
+    yum_repository:
+      name: rabbitmq
+      description: rabbitmq
+      baseurl: "https://dl.bintray.com/rabbitmq/rpm/rabbitmq-server/v3.7.x/el/{{ansible_distribution_major_version}}"
+      gpgcheck: yes
+      gpgkey: https://dl.bintray.com/rabbitmq/Keys/rabbitmq-release-signing-key.asc
```
Comment (jaredledvina) [inline tasks/CentOS/rabbit.yml:13]: Interesting, Ansible keeps failing to get this in the Travis build:
```
TASK [role_under_test : Ensure the RabbitMQ APT repo GPG key is present] *******
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to download key at https://www.rabbitmq.com/rabbitmq-release-signing-key.asc: Connection failure: ('The read operation timed out',)"}
	to retry, use: --limit @/etc/ansible/roles/role_under_test/tests/test.retry
```

but this seems fine to me, I'll see if I can repro these timeout issues, might just be something on the Travis side.

Comment (romainrbr) [reply tasks/CentOS/rabbit.yml:13]: If I'm not mistaken the issue is actually with the Ubuntu / Debian tasks on Travis isn't it ?

```
TASK [role_under_test : Ensure the RabbitMQ APT repo GPG key is present] *******
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to download key at https://www.rabbitmq.com/rabbitmq-release-signing-key.asc: Connection failure: ('The read operation timed out',)"}
	to retry, use: --limit @/etc/ansible/roles/role_under_test/tests/test.retry
```

This is part of `cmacrae.sensu/tasks/Ubuntu/rabbit.yml` & `cmacrae.sensu/tasks/Debian/rabbit.yml`
The CentOS build seems to be passing without problems : https://travis-ci.org/sensu/sensu-ansible/jobs/335130379

Comment (jaredledvina) [reply tasks/CentOS/rabbit.yml:13]: Woooow yeah, I totally missed that. Not related to your change at all. Let me try to unstick them.

Comment (jaredledvina) [reply tasks/CentOS/rabbit.yml:13]: Working on it, looks like `https://www.rabbitmq.com/rabbitmq-release-signing-key.asc` is being super flakey. Sometime's it pulls instantly, other times, it lags a ton. It also looks like Debian/Ubuntu are using the older rabbitmq mirrors not PackageCloud/Bintray so, that's not ideal either.

**Review by jaredledvina — APPROVED** [review]
Boom! Looks legit and the build passes (just had to retry that a few times for ya)

**Pull request merged**